swift-sdk/IterableAPI.swift

```diff
@@ -204,6 +204,16 @@ public final class IterableAPI: NSObject {
         return internalImplementation.inAppManager
     }
     
+    public static var flexMessagingManager: IterableFlexMessagingManagerProtocol {
```

Comment:
Should provide comments to align with inAppManager. Im sure this will be implemented in the future.

Reply:
Yep.